doc/vxlan/data_plane_VXLAN_HLD.md

```diff
@@ -0,0 +1,510 @@
+    }
+}
+
+##### VXLAN_REMOTE_VTEP_VNI
+
+container VXLAN_REMOTE_VTEP_VNI {
+
+    description "config db VXLAN_REMOTE_VTEP_VNI table";
+
+    list VXLAN_REMOTE_VTEP_VNI_LIST {
+        key "name mapname";
+
+        leaf name {
+            type leafref {
+                path "/svxlan:sonic-vxlan/svxlan:VXLAN_REMOTE_VTEP_LIST/svxlan:VXLAN_REMOTE_VTEP_VNI_LIST/svxlan:name";
+            }
+        }
+
+        leaf mapname {
+            type string;
+        }
+
+        leaf vni {
+            mandatory true;
+            type stypes:vnid_type;
+        }
+    }
+}
+
+
+#### Config DB Enhancements
+
+##### 2.1 VXLAN_REMOTE_VTEP_TABLE
+
+Producer: config manager
+
+Consumer: VxlanMgr
+
+Description: This table holds the tunnel name specifying the VTEP remote IP used for data plane configured tunnels.
+
+; New table
+
+key = VXLAN_REMOTE_VTEP_TABLE|remote_vtep_name
+                          ; VTEP name as a string
+; field = value
+src_ip = ipv4_address ; Source Ipv4 address of the remote VTEP.
+
+
+##### 2.2 VXLAN_REMOTE_VTEP_VNI_TABLE
+
+Producer: config manager
+
+Consumer: VxlanMgr
+
+Description: This table specifies mapping of remote VTEP to VNI.
+
+Schema:
+
+; New table
+; specifies mapping of remote VTEP to VNI
+
+key = VXLAN_REMOTE_VTEP_VNI_TABLE|remote_vtep_name|tunnel_map_name
+                  ; Remote VTEP name + tunnel map name
+; field = value
+vni = 1*8DIGIT    ; vni that remote VTEP is member of.
+
+#### 3. APP_DB Enhancements
+
+##### 3.1 VXLAN_DATAPLANE_VTEP_TABLE
+
+Producer: config manager
+
+Consumer: VxlanMgr
+
+Description: This table holds the tunnel name specifying local VTEP's source IP used for Dataplane configured tunnels.
+
+Schema:
+
+;New table
+;holds the tunnel name specifying the VTEP source IP used for Dataplane configured tunnels
+
+key = VXLAN_DATAPLANE_VTEP:nvo_name
+                          ; nvo or VTEP name as a string
+; field = value
+source_vtep = tunnel_name
+
+
+##### 3.2 VXLAN_DATAPLANE_VTEP_VNI_TABLE
+
+Producer: config manager
+
+Consumer: VxlanMgr
+
+Description: This specifies mapping of remote VTEP to VNI.
+
+Schema:
+
+; New table
+; specifies mapping of remote VTEP to VNI
+
+key = VXLAN_DATAPLANE_VTEP_VNI:remote_vtep_name:map_name
+                  ; Remote VTEP name + map name
+; field = value
+vni = VNI id
+
+
+### 4. Modules Design and Flows
+
+Following configurations are required for enabling EVPN L2 service on SONiC device:
+
+Create VXLAN data plane tunnel.
+Configure L2VNI-VLAN mapping for the given NVO.
+Configure control plane with remote VTEPS and bind them to the same vni as source VTEP.
+
+#### 4.1 The corresponding CONFIG_DB entries are as follows
+
+VLAN|Vlan{{vlan_id}}
+    "vlanid" : {{vlan_id}}
+
+VXLAN_TUNNEL_TABLE|{{source_vtep_name}}
+    "src_ip" : {{ipv4_address}}
+
+VXLAN_TUNNEL_MAP|{{source_vtep_name}}|{{tunnel_map_name}}
+    "vni" : {{vni_id}}
+    "vlan" : {{vlan_id}}
+
+VXLAN_REMOTE_VTEP_TABLE|{{remote_vtep_name}}
+    "src_ip" : {{ipv4_address}}
+
```

Comment:
this can be refered as rempte ip  ?

Reply:
you mean change the name from "src_ip" to "remote_ip" ?